insights/parsers/sys_fs_cgroup_memory.py

```diff
@@ -0,0 +1,45 @@
+"""
+SysFsCgroupMemory - Datasource ``sys_fs_cgroup_memory``
```

Comment:
Please update the title accordingly.

You may want to make the top title a bit rough, in case we can add other parsers related to "/sys/fs/cgroup/memory" into this file.